### PR TITLE
Feature/sensors

### PR DESCRIPTION
**Pull request by orangecms**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix
- [x] Feature implementation
- [x] Doc updates

* **What changes have you introduced?**

#611, #432

* **Does this PR introduce a breaking change?**

no

* **Preview / Steps to verify your work**:

### CODE REVIEW

**Comment by orangecms** [comment]
The main part is done now. Feedback would be great. :)

The Codacy feedback is a bit surprising. Both `prettier` and `eslint` are fine with the current state.

File: scripts/sensors.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import threading
```
Comment (bessman) [inline scripts/sensors.py:1]: Style: imports should be sorted alphabetically.

File: scripts/bridge.py [hunk 2]
```diff
@@ -182,6 +183,10 @@ def main():
             angle4 = parsed_stream_data['angle4']
             robotic_arm.setServo(angle1, angle2, angle3, angle4)
 
+        # ---------------------------- Sensors block ------------------------------
```
Comment (bessman) [inline scripts/bridge.py:186]: "Sensors block" text looks misaligned with "Robtic Arm block" and "Write block". Move it a little to the right.
(While you're editing this part of the code anyway, you could also fix the spelling of "Robtic")

Comment (orangecms) [reply scripts/bridge.py:186]: I'll do that separately to scope the PR. Thanks for the hint! Indentation is a thing... The Python and JavaScript parts are different, I guess. :sweat_smile:

File: scripts/sensors.py (hunk 1 above)
Comment (orangecms) [reply scripts/sensors.py]: Fixed. To be honest, I just copied them from another file. I guess they can use some cleanup then.

**Comment by mariobehling** [comment]
Lets move forward and follow up on Codacy later. Merging this therefore. Well-done!

**Comment by mariobehling** [comment]
@orangecms Usually we ask contributors to fork the repository and work in their own fork and then make a PR to the repo. One reason is that we had a lot of abandoned branches in the past and it became more and more difficult to keep track of PRs. People come and go and some people make a PR and dont follow up on change requests. This kind of policy helped us to stay focused. Thanks

**Comment by orangecms** [comment]
Okay cool, will do from now on.

How is release engineering handled? With a new feature, a minor bump would make sense, right? That would be great for distributions so they can pick up the release from a tag and upgrade packages. I can take care of that if that's okay.

**Comment by mariobehling** [comment]
That would be awesome. Thank you!

**Pull request merged**